woocommerce/woocommerce
# Update listing content

Adding changes to improve search discovery for important ecommerce-related keywords such as 'product pages' and 'online store'.

### All Submissions:


### How to test the changes in this Pull Request:

1. Check that text changes are correct (no incorrect spelling, punctuation or spacing)
2. Check that the hyperlinks are correct and functioning correctly.
